### PR TITLE
Minor README Updates for Installation

### LINKED ISSUE #1010: [FEATURE]: Improve README Instructions
### Feature Name

Improve README Instructions

### Your Name

Faris Durrani

### Description

We attach a few installation recommendations from today's new member meeting:
1. What you should be seeing if the installation runs correctly
1. Suggesting restart after installing WSL
1. Adding NVM to PATH on Windows after installing
1. Adding GitHub Student Dev Packet link

### PR DESCRIPTION
**Pull request by farisdurrani**
# Minor README Updates for Installation

Github Issue Number Here: #1010 
**What user problem are we solving?**
Early in the semester, some new members had difficulty installing and starting the backend and frontend. We make slight improvements to the README, especially in adding screenshots, to help with this process.

**What solution does this PR provide?**
Upgraded Node version, added screenshots, linked to Bug Manual page, fixed build issue

**Testing Methodology**
> How did you test your changes and verify that existing functionality is not broken

While no testing is necessary since changes are to the README, starting the frontend showed no errors.

**Any other considerations**
Some items on the original Issue were not considered important enough to be added to the README, and hence they're put to the [Bug Manual page](https://github.com/DSGT-DLP/Deep-Learning-Playground/wiki/Bug-Manual), now linked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 
 # Getting Started
 
 ### Prerequisites
 
 Have the following installed first:
 
-1. [Node.js v18 via NVM](https://github.com/nvm-sh/nvm#installing-and-updating) (Install nvm first, and then install node & npm using nvm)
+1. [Node.js v20 via NVM](https://github.com/nvm-sh/nvm#installing-and-updating) (Install nvm first, and then install node & npm using nvm)
 1. [Yarn 1.x](https://classic.yarnpkg.com/lang/en/docs/install) (Must be installed after npm. May upgrade to Yarn Modern in the future, keep an eye out for that!)
```
Comment (karkir0003) [inline README.md]: do we need people to install yarn even though we are on pnpm?

File: README.md [hunk 2]
```diff
@@ -37,13 +38,20 @@ Have the following installed first:
    1. [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker)
    1. [Go](https://marketplace.visualstudio.com/items?itemName=golang.Go)
 
-## Clone the Repository
+## 1. Clone the Repository
 
-`git clone https://github.com/DSGT-DLP/Deep-Learning-Playground.git` in the directory of your choice. If this installation is for the beginner project, do `git clone https://github.com/DSGT-DLP/dlp-practice.git`.
+If you want the main project, run in the directory of your choice:
```
Comment (karkir0003) [inline README.md]: bold the word "main"

File: README.md [hunk 3]
```diff
@@ -37,13 +38,20 @@ Have the following installed first:
...
+```sh
+git clone https://github.com/DSGT-DLP/Deep-Learning-Playground.git
+```
+
+If this installation is for the beginner project, instead do:
```
Comment (karkir0003) [inline README.md]: bold "for the beginner project"

**Review by karkir0003 — CHANGES_REQUESTED** [review]
left minor comments. looks good to me overall. thanks for this faris

File: README.md (hunk 1 above)
Comment (farisdurrani) [reply README.md]: Good point. Removed

**Comment by farisdurrani** [comment]
Resolved comments @karkir0003. Thanks!

**Review by karkir0003 — APPROVED** [review]

**Comment by karkir0003** [comment]
approved @farisdurrani

**Pull request merged**